td-agent/debian/td-agent.lintian-overrides

```diff
@@ -13,3 +13,6 @@ td-agent: wrong-path-for-interpreter
 td-agent: duplicate-font-file
 td-agent: jar-not-in-usr-share
 td-agent: incorrect-libdir-in-la-file
+td-agent: unusual-interpreter opt/td-agent/lib/ruby/gems/2.4.0/gems/bundler-1.16.6/lib/bundler/templates/Executable
```

Comment:
Is there a way to use wildcard for such version numbers?

Reply:
I'' try to use wildcard.

Reply:
Umm, It's strange wildcard behavior.

```
opt/td-agent/lib/ruby/gems/*/gems/bundler-*/lib/bundler/templates/Executable NG
opt/td-agent/lib/ruby/gems/*/gems/bundler-1.16.6/lib/bundler/templates/Executable NG
opt/td-agent/lib/ruby/gems/2.4.0/gems/bundler-*/lib/bundler/templates/Executable NG
*/lib/bundler/templates/Executable NG
```

Reply:
At least 2.5.0 should work as expected.
https://nthykier.wordpress.com/2011/05/10/lintian-2-5-0-overrides-and-other-changes/

Reply:
Wildcard in overrides no longer appears to work mid-string
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=817068

It will not affect us.